app/models/miq_server/role_management.rb

```diff
@@ -165,6 +158,17 @@ def role=(val)
       end
     end
 
+    roles
+  end
+
+  def role
+    server_role_names.join(',')
```

Comment:
I wonder if this should be sorted, like you're sorting above ^ or if we should sort the result in the tests below...
We could have unsorted values in the existing configuration, which is fine, but I noticed your test wasn't sorting.

Reply:
Doesn't the `server_role_names` accessor already sort the roles [here](https://github.com/ManageIQ/manageiq/blob/c686ae46c0ec7e6284a38402241e40e3a5c288a5/app/models/miq_server/role_management.rb#L132), and the `server_role_names=` setter [here](https://github.com/ManageIQ/manageiq/blob/c686ae46c0ec7e6284a38402241e40e3a5c288a5/app/models/miq_server/role_management.rb#L142)

Reply:
ooh, it does.  Sweet.  Move along me, nothing to see here.